velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -101,6 +104,44 @@ struct HiveConnectorSplit : public connector::ConnectorSplit {
         properties(_properties),
         rowIdProperties(_rowIdProperties) {}
 
+  HiveConnectorSplit(
```

Comment:
Why introduce a new ctor? This is for backward compatibility with Prestissimo? If it is the case, can we use backward macro for that? Thanks!